web_api/app/precomputed_annotations.py

```diff
@@ -65,8 +67,8 @@ async def add_multiple(
     resolution: Annotated[tuple[float, float, float], Query()],
```

Comment:
Same for the write -- should we allow the start and end coord to be given in resolutions that are not the same as the ones used for annotations?

Reply:
Same answer.  I don't have strong feelings about it, but slightly prefer to have only one resolution.